### PR TITLE
Update Logging functions

### PR DESCRIPTION
**Pull request by catherine-martlin**
First attempt at a decorator to log timing. Defintiely useful for functions, possibly also queries.

### CODE REVIEW

**Comment by catherine-martlin** [comment]
@bourque - This is currently just updated to add in a function that will allow people to wrap a module or function in their code and return a runtime for it. I'll create documentation on how to use it shortly.

**Comment by bourque** [comment]
@catherine-martlin Sounds good.  We will do a review once the documentation is ready.

**Comment by bourque** [comment]
On second thought, I am going to review this now and we can merge this in before we merge in #258 (so that we can describe the existence of this decorator in the guidelines themselves).

File: jwql/utils/logging_functions.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 Authors
 -------
 
-    - Catherine Martlin 2018
+    - Catherine Martlin 2018, 2019
```
Comment (bourque) [inline jwql/utils/logging_functions.py]: @catherine-martlin Actually, could you remove the years altogether here?  We haven't been including years in our other module docstrings, so removing them would be consistent.

File: jwql/utils/logging_functions.py [hunk 2]
```diff
@@ -243,3 +243,34 @@ def wrapped(*a, **kw):
...
+        func()
+        t2_cpu = time.process_time()
+        t2_time = time.time()
+
+        # Log execution time
+        hours_cpu, remainder_cpu = divmod(t2_cpu - t1_cpu, 60 * 60)
+        minutes_cpu, seconds_cpu = divmod(remainder_cpu, 60)
+        hours_time, remainder_time = divmod(t2_time - t1_time, 60 * 60)
+        minutes_time, seconds_time = divmod(remainder_time, 60)
+        logging.info('Elapsed Real Time of function: {0:.0f}:{1:.0f}:{2:f}'.format(hours_time, minutes_time, seconds_time))
```
Comment (bourque) [inline jwql/utils/logging_functions.py]: @catherine-martlin We could slightly improve this by using:

```python
logging.info('Elapsed Real Time of {}: {0:.0f}:{1:.0f}:{2:f}'.format(func.fun_name, hours_time, minutes_time, seconds_time))
```

**Comment by bourque** [comment]
Thanks @catherine-martlin, looks great!

**Pull request merged**